.husky/pre-commit

```diff
@@ -23,6 +23,15 @@ root_path=$(git rev-parse --show-toplevel)
 # List the directories to check for changes
 directories=("front" "connectors")
 
+# Flag to check if docs:check should be run
+run_docs_check=false
+
+# Check if any files in front/pages/api/v1 have changed
+if git diff --cached --name-only | grep -qE "^front/pages/api/v1/"; then
```

Comment:
Is this working if the git commit is done from a different place than the root of the repo ?

Reply:
Yeah not sure, I updated